pkg/network/network.go

```diff
@@ -17,15 +17,22 @@ limitations under the License.
 package network
 
 import (
+	"context"
 	"fmt"
+	"net"
+	"os"
+	"strconv"
 	"strings"
 
+	"github.com/GoogleCloudPlatform/lustre-csi-driver/pkg/k8sclient"
 	"github.com/vishvananda/netlink"
 	"k8s.io/klog/v2"
 )
 
 const (
 	multiRailLabel = "lustre.csi.storage.gke.io/multi-rail"
+	// lustreInstanceIP is the IP of the Lustre instance passed in through volumeContext.
+	lustreInstanceIP = "10.170.0.0/16"
```

Comment:
Instance IP is dynamic, not constant. It can only be retrieved from volumeContext during NodeStageVolume.